openfecwebapp/templates/partials/candidate/individual-contributions.html

```diff
@@ -0,0 +1,152 @@
+{% import 'macros/cycle-select.html' as select %}
+
+<section id="section-5" role="tabpanel" aria-hidden="true" aria-labelledby="section-5-heading">
+  <div class="heading--section">
+    <h2 id="section-5-heading">Individual contributions</h2>
+  </div>
+
+  <div class="slab slab--inline slab--neutral u-padding--left u-padding--right">
+
+    {{ select.candidate_cycle_select(cycles, cycle, id="cycle-5") }}
+
+  <span class="t-sans t-bold">Data is included from these committees:</span>
+
+  <ul class="list--bulleted">
+    {% for committee in committee_groups['P'] | reverse %}
+    <li>
+      <a class="t-sans" href="{{ url_for('committee_page', c_id=committee.committee_id, cycle=cycle) }}">{{ committee.name }}</a>
+    </li>
+    {% endfor %}
+    {% for committee in committee_groups['A'] | reverse %}
+    <li>
+      <a class="t-sans" href="{{ url_for('committee_page', c_id=committee.committee_id, cycle=cycle) }}">{{ committee.name }}</a>
+    </li>
+    {% endfor %}
+  </ul>
+
+  <div class="entity__figure row">
+
+    <div class="content__section">
+
+      <h3>Individual contributions by transaction</h3>
+
+      <div class="row">
+        <fieldset class="toggles">
+          <legend class="label">View by:</legend>
+
+          <a class="u-float-right button--alt button--browse"
+              href="{{ url_for(
+                'individual_contributions',
+                two_year_transaction_period=cycle,
+                min_date=cycle_start(cycle) | date(fmt='%m-%d-%Y'),
+                max_date=cycle_end(cycle) | date(fmt='%m-%d-%Y')
+              ) }}{% for id in committee_ids %}&committee_id={{ id }}{% endfor %}">Filter this data</a>
+
+          <label for="toggle-state">
+            <input id="toggle-state" type="radio" class="panel-toggle-control" name="individual-contributions" value="contributor-state" checked />
+            <span class="button--alt">Contributor state</span>
+          </label>
+          <label for="toggle-size">
+            <input id="toggle-size" type="radio" class="panel-toggle-control" name="individual-contributions" value="contribution-size" />
+            <span class="button--alt">Contribution size</span>
+          </label>
+          <label for="toggle-all">
+            <input id="toggle-all" type="radio" class="panel-toggle-control" name="individual-contributions" value="all-transactions" />
+            <span class="button--alt">All transactions</span>
+          </label>
+        </fieldset>
+      </div>
+
+      <div id="contributor-state" class="panel-toggle-element">
+
+        <div class="results-info results-info--simple">
+          <ul class="u-float-left">
+            <li class="tag__category">
+              <div class="tag__item">Report year: {{cycle - 1}} to {{cycle}}</div>
+            </li>
+          </ul>
+
+          <button type="button" class="js-export button button--cta button--export" data-export-for="contributor-state">Export</button>
```

Comment:
I was confused at first, but then realized that it wasn't disabling the button because I didn't label the button correctly (individual-contributions).